tools/dependencies/known-dependencies.txt

```diff
@@ -732,4 +732,6 @@ zookeeper-jute-3.5.9.jar
 zstd-jni-1.3.3-1.jar
 zstd-jni-1.4.3-1.jar
 jakarta.activation-api-1.2.1.jar
-jakarta.xml.bind-api-2.3.2.jar
\ No newline at end of file
+jakarta.xml.bind-api-2.3.2.jar
+postgresql-42.3.3.jar
+checker-qual-3.5.0.jar
```

Comment:
It's better to use one version if needed, you can pin the version in the root pom (dependency management)

Reply:
The sub-dependencies of postgresql-42.3.3 include checker-qual-3.5.0.

Reply:
I know, but you can try it, we always hope for one version.